src/components/ContainerHeader.vue

```diff
@@ -183,9 +196,35 @@ export default {
 .kiwi-header-options {
     display: inline-block;
     float: right;
-    margin-top: 4px;
-    margin-right: 4px;
 }
+
+.kiwi-header-topic{
+    display: inline-block;
+}
+
+.kiwi-header-options .u-button{
+    text-transform: uppercase;
+    font-size: 0.7em;
+    font-weight: 600;
+    letter-spacing: 0.2em;
+    padding: 0.5em 1.7em;
+    line-height: 2em;
+    border-radius: 0.4em;
+}
+
+.kiwi-header .kiwi-header-join-channel-button{
```

Comment:
This full selector is now redundant. It can just be `.kiwi-header-join-channel-button`

Reply:
#b384959 :)